src/components/Login/LoginContainer.tsx

```diff
@@ -0,0 +1,150 @@
+import { styled } from 'styled-components';
+import { useState } from 'react';
+
+const LoginWrapper = styled.form`
+display: flex;
+padding: 88px 20px 124px 20px;
+flex-direction: column;
+align-items: center;
+gap: 10px;
+`;
+const IsLoginBtn = styled.button`
+display: flex;
+height: 48px;
+justify-content: center;
+align-items: center;
+align-self: stretch;
+border-radius: 12px;
+border:none;
+background: #0147C8;
+color: #FFF;
+font-family: "SF Pro";
+font-size: 15px;
+font-style: normal;
+font-weight: 510;
+line-height: normal;
+letter-spacing: -0.6px;
+cursor:pointer;
+`;
+const KakaoBtn = styled.button`
+display: flex;
+height: 48px;
+justify-content: center;
+align-items: center;
+align-self: stretch;
+border-radius: 12px;
+border:none;
+background: #F7E600;
+color:#3A1D1D;
+font-family: "SF Pro";
+font-size: 15px;
+font-style: normal;
+font-weight: 510;
+line-height: normal;
+letter-spacing: -0.6px;
+cursor:pointer;
+`;
+const IdInput = styled.input`
+height: 48px;
+align-self: stretch;
+flex-shrink: 0;
+border-radius: 12px;
+background: #F3F3F5;
+border:none;
+&::placeholder {
+ padding-left:10px;
+ color:#B9B9B9;
+}
+&:focus{
+  background: #EBF2FF;
+  outline: none;
+}
+margin-top:20px;
+cursor:pointer;
+`;
+
+const PwdInput = styled.input`
+align-self: stretch;
+height: 48px;
+flex-shrink: 0;
+border-radius: 12px;
+background: #F3F3F5;
+border:none;
+&::placeholder {
+  padding-left:10px;
+  color:#B9B9B9;
+ }
+ &:focus{
+  background: #EBF2FF;
+  outline: none;
+  cursor:pointer;
+}
+`;
+const BtnWrapper = styled.div`
+display:flex;
+flex-direction:column;
+align-items: center;
+width:100%;
+gap:10px;
+margin-top:50px;
+`;
+const Explain = styled.div`
+color: #CFCFCF;
+text-align: center;
+font-family: "SF Pro";
+font-size: 13px;
+font-style: normal;
+font-weight: 400;
+line-height: 19px; /* 146.154% */
+letter-spacing: -0.52px;
+margin-top:60px;
+`;
+export default function LoginContainer() {
+  const RestAPIKey = '641457a0e25cbce8346a2c6c48d7179a';
+  const RedirectURL = 'http://localhost:4000/auth/kakao/callback';
+  const KakaoURL = `https://kauth.kakao.com/oauth/authorize?client_id=${RestAPIKey}&redirect_uri=${RedirectURL}&response_type=code`;
+  const handleLogin = () => {
+    window.location.href = KakaoURL;
+  };
+  const [Id, setId] = useState('');
+  const [Password, setPassword] = useState('');
```

Comment:
네이밍 컨벤션이 이상하네요~
카멜케이스로 작성되어야합니다!
```suggestion
  const [id, setId] = useState('');
  const [password, setPassword] = useState('');
```